src/main/java/io/github/wimdeblauwe/errorhandlingspringbootstarter/handler/AbstractGraphqlExceptionHandler.java

```diff
@@ -0,0 +1,39 @@
+package io.github.wimdeblauwe.errorhandlingspringbootstarter.handler;
+
+import io.github.wimdeblauwe.errorhandlingspringbootstarter.GraphqlExceptionHandler;
+import io.github.wimdeblauwe.errorhandlingspringbootstarter.mapper.ErrorCodeMapper;
+import io.github.wimdeblauwe.errorhandlingspringbootstarter.mapper.ErrorMessageMapper;
+import io.github.wimdeblauwe.errorhandlingspringbootstarter.mapper.GraphQlErrorMapper;
+import io.github.wimdeblauwe.errorhandlingspringbootstarter.mapper.HttpStatusMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+
+public abstract class AbstractGraphqlExceptionHandler implements GraphqlExceptionHandler {
+    protected final HttpStatusMapper httpStatusMapper;
```

Comment:
I guess these can be `private` since there are the getter methods anyway?

Reply:
We can mark them as private, yes. I also mostly used the specialized "get...(Throwable exception)" methods.